### DISCUSSION TITLE
About compatible

### DISCUSSION THREAD

**Discussion by cloudandzack007** [OpenDDS/OpenDDS#3879] [Q&A] [answered]
Hello，if the dds 3.21 and dds 3.13.2 is compatible？if a app with 3.21，another is 3.13.2，if they will connect successful or not。

**Comment by jwillemsen** [comment, marked as answer]
On Linux we normally use the `gnuace` type with `mwc.pl`, can you try that?

**Reply by cloudandzack007** [reply]
gnuace can work，thank you。
can you answer the new question about compitable

**Reply by cloudandzack007** [reply]
> gnuace can work，thank you。
> can you answer the new question about compatible

**Comment by mitza-oci** [comment]
Please don't edit a question to make it a completely different question, it's hard to follow the conversation.

> Hello，if the dds 3.21 and dds 3.13.2 is compatible？if a app with 3.21，another is 3.13.2，if they will connect successful or not。

If you're using the DDSI-RTPS implementations of both the Discovery and Transport mechanisms then there's a good chance they can work together.  Please test it and let us know.  

As I wrote in the other thread, I don't recommend using 3.13.x at all since there's been a lot of progress in OpenDDS development over the last four+ years.
